decidim-core/lib/decidim/spring.rb

```diff
@@ -14,3 +14,4 @@ def start
 end
 
 Spring::Watcher::Listen.prepend Decidim::SpringWatcher
+Spring.application_root = ""
```

Comment:
Why do you want to change spring settings here?

I believe we should not change Spring settings in this PR. If there is some actual reason for this change, please create a separate PR for such configuration changes.